### ISSUE TITLE
Video upload fail when user lack organisation

### ISSUE THREAD

**Issue by petterreinholdtsen** [Frikanalen/frikanalen#182]
A new user had problem uploading a video file.  I sat down with her and we discovered that the JavaScript console reported that the creation of the video entry was rejected because the user had no associated organization.  This seem like an oversight, as non-member users should be able to upload videos too.

**Comment by toresbe** [comment]
This is correct behavior but with awful UX and error handling. The compromise solution we have landed on is that non-member users should have an "organization" which is registered as a private person. I will bear this issue in mind and close when the new upload dialog and signup flow addresses these issues. Thanks for reporting.
